recipes/ipyflow-core/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "ipyflow-core" %}
+{% set version = "0.0.176" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ipyflow-core-{{ version }}.tar.gz
+  sha256: ff7c5fef42472a6ca867166b0d0ea2af84cdb9b1b93e727586d4ad16ade60d55
+
+build:
+  skip: true  # [py2k]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python
+    - setuptools >=48
+    - wheel >=0.30.0
+    - setuptools-git-versioning
+    - pip
+  run:
+    - python
+    - astunparse
+    - black
+    - contextvars  # [py<37]
```

Comment:
Assuming you don't need Python 3.7 support I would opt for removing this and depending on `python >=3.8`:
```suggestion
```